e2eAIOK/common/trainer/data/data_builder_cv.py

```diff
@@ -3,19 +3,32 @@
 from torchvision.datasets.folder import ImageFolder
 from timm.data.constants import IMAGENET_DEFAULT_MEAN, IMAGENET_DEFAULT_STD
 from timm.data import create_transform
+import e2eAIOK.common.trainer.utils.extend_distributed as ext_dist
+
 
 from e2eAIOK.common.trainer.data_builder import DataBuilder
 
 class DataBuilderCV(DataBuilder):
     def __init__(self, cfg):
         super().__init__(cfg)
-
+    
     def prepare_dataset(self):
         """
             prepare CV related dataset
         """
-        raise NotImplementedError("Please implement specific dataset builder")
-
+        if self.cfg.data_set in ["CIFAR10","CIFAR100"]:
+            dataset_train = self.build_dataset(is_train=True, cfg=self.cfg)
+            dataset_val = self.build_dataset(is_train=False, cfg=self.cfg)
+        else:
+            raise RuntimeError(f"dataset {self.cfg.data_set} not supported")
+        return dataset_train, dataset_val
+    
+    def build_dataset(self):
+        transform = build_transform(is_train,cfg)
+        if cfg.data_set == 'CIFAR10':
+            dataset = datasets.CIFAR10(cfg.data_path, train=is_train, transform=transform, download=True)
+        elif cfg.data_set == 'CIFAR100':
+            dataset = datasets.CIFAR100(cfg.data_path, train=is_train, transform=transform, download=True)
```

Comment:
for multi dataset, create data_builder_cifar10/cifar100 to support different datasets, and put common functions for CV in data_builder_cv

Reply:
@Peach-He , the above comment is a todo or an explaination?

Reply:
I think it don't need to provide many files separately for cifar10 and cifar100, it only need four lines to build the dataset.

Reply:
> I think it don't need to provide many files separately for cifar10 and cifar100, it only need four lines to build the dataset.

will image1K and other dataset will be supported in future? If so, I think using a separate class is making sense

Reply:
Got it. I will create a separate class for cifar dataset.

Reply:
Resolved.